catalog/ui/catalog-ui-search-api/pom.xml

```diff
@@ -38,6 +38,16 @@
             <artifactId>platform-util</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.codice.thirdparty</groupId>
+            <artifactId>geotools-suite</artifactId>
+            <version>${org.geotools.bundle.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sparkjava</groupId>
+            <artifactId>spark-core</artifactId>
+            <version>${spark.version}</version>
+        </dependency>
```

Comment:
❓ Are these absolutely necessary dependencies for the API? I know we use some Geotools interfaces but does that require the entire suite? Also, what Spark classes are we using in our APIs? If possible we should write the interfaces such that none of these (with the exception of the core Geotools interfaces) are used (but they're fine in the impl).